.circleci/config.yml

```diff
@@ -124,7 +124,7 @@ jobs:
 
   testjdk8:
     docker:
-      - image: circleci/openjdk:11.0.6-jdk-stretch
```

Comment:
This was previously left behind because we needed a stretch image which has both JDK 8 in base for running tests, and JDK 11 for. compile